isodesign/process/calculation.py

```diff
@@ -412,19 +378,48 @@ def read_tvar_sim_files(self, path_folder):
 
         for files_path in tvar_sim_paths:
             output_data = pd.read_csv(files_path, sep="\t")
-            self.dict_output_dataframe.update({"Flux_name": output_data["Name"]})
-            self.dict_output_dataframe.update({"Kind": output_data["Kind"]})
-            self.dict_output_dataframe.update({"Value": output_data["Value"]})
-            self.dict_output_dataframe.update({"Value_difference": imported_tvar["Value"] - output_data["Value"]})
-            self.dict_output_dataframe.update({f"{files_path.name}_SD": output_data['SD']})
-        logger.debug(f"dict_output_dataframe: {self.dict_output_dataframe}")
+            self.dict_summarize_tvar.update({"Flux_name": output_data["Name"]})
+            self.dict_summarize_tvar.update({"Kind": output_data["Kind"]})
+            self.dict_summarize_tvar.update({"Value": output_data["Value"]})
+            self.dict_summarize_tvar.update({"Value_difference": self.get_initial_tvar_files() - output_data["Value"]})
+            self.dict_summarize_tvar.update({f"{files_path.name}_SD": output_data['SD']})
+        logger.debug(f"dict_summarize_tvar: {self.dict_summarize_tvar}")
```

Comment:
`dataframes = [pd.read_csv(files_path, sep="\t") for files_path in tvare_sim_paths]`
`summary = pd.concat(dataframes)`